bookkeeper-server/src/main/java/org/apache/bookkeeper/proto/PacketProcessorBase.java

```diff
@@ -66,7 +66,11 @@ protected boolean isVersionCompatible() {
     }
 
     protected void sendResponse(int rc, Object response, OpStatsLogger statsLogger) {
-        channel.writeAndFlush(response, channel.voidPromise());
+        if (channel.isActive()) {
+            channel.writeAndFlush(response, channel.voidPromise());
+        } else {
+            LOGGER.info("Netty channel is inactive, hence bypassing netty channel writeAndFlush during sendResponse");
```

Comment:
please report the channel(), it will bring useful information about the client that won't receive the response.
please use "debug" and not "info"

Reply:
Changed it. As an example, it prints like this: `Netty channel [id: 0x8f9a1f37, L:/192.168.160.3:3181 ! R:/192.168.160.1:61752] is inactive, hence bypassing netty channel writeAndFlush during sendResponse`